src/App.php

```diff
@@ -78,17 +78,19 @@ private function getEventManager()
     /**
      * Application execution.
      *
-     * @param mixed|null $request  Representation of an outgoing, client-side request.
-     * @param mixed|null $response Representation of an incoming, server-side response.
+     * @param mixed|null $request  Representation of an outgoing,
+     *  client-side request.
+     * @param mixed|null $response Representation of an incoming,
+     *  server-side response.
      *
      * @return mixed
      */
     public function run($request = null, $response = null)
     {
-        $request = $request ?: ServerRequestFactory::fromGlobals();
-        $response = $response ?: new Response();
+        $request = $request ?: $this->getContainer()->get("request");
+        $response = $response ?: $this->getContainer()->get("response");
```

Comment:
use `$this->container` instead of `$this->getContainer()` for internally call, as I cleaned up on previous PR